nobrainer/bayesian_utils.py

```diff
@@ -23,8 +23,13 @@ def default_loc_scale_fn(
     untransformed_scale_constraint=None,
     weightnorm=False,
 ):
+    """
```

Comment:
Could we make sure all the docstrings are formatted similar to other scientific python packages for example https://www.tensorflow.org/community/contribute/docs_ref

Reply:
Sure, i will update these accordingly. Thanks for pointing @satra.